src/main/java/racingcar01/domain/Car.java

```diff
@@ -0,0 +1,36 @@
+package racingcar01.domain;
+
+public class Car {
+    private static final int DEFAULT_DISTANCE = 0;
+
+    private final Name name;
+    private final Distance distance;
+
+    private Car(Name name) {
+        this(name, Distance.of(DEFAULT_DISTANCE));
+    }
+
+    private Car(Name name, Distance distance) {
+        this.name = name;
+        this.distance = distance;
+    }
+
+    public static Car of(String name) {
+        return new Car(Name.of(name));
+    }
+
+    public Car movedCar(MoveStrategy move) {
+        if (move.isMoveable()) {
+            return new Car(name, distance.add());
+        }
+        return this;
+    }
+
+    public String name() {
+        return name.name();
+    }
+
+    public int distance() {
+        return distance.distance();
+    }
```

Comment:
원시값을 포장했으면
name.name(), distance.distance() 보다는
```suggestion
    public String name() {
        return name.toString();
    }

    public int distance() {
        return distance.toInt();
    }
```
같은 메서드명으로 가져오는게 좋지 않을까?
toInt, toString, value 등